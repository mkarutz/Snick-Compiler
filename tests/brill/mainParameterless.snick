proc main(val int x)
    write x;
end
