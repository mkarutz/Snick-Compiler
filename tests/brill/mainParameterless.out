Failure: Main procedure must be parameterless.
Compilation Failed
